packages/plugin-ext/src/plugin/webview-views.ts

```diff
@@ -186,6 +187,19 @@ export class WebviewViewExtImpl implements theia.WebviewView {
         }
     }
 
+    get badge(): theia.ViewBadge | undefined {
+        this.assertNotDisposed();
+        return this._badge;
+    }
+
+    set badge(badge: theia.ViewBadge | undefined) {
+        this.assertNotDisposed();
+        if (this._badge !== badge) {
+            this._badge = badge;
+            this.proxy.$setBadge(this.handle, badge);
```

Comment:
Same as above.

Reply:
This has been adressed also in the new version, with same adaptation due to `badge` being potentially `undefined`.